addons/LPCAnimatedSprite/LPCAnimatedSprite2D.gd

```diff
@@ -147,8 +162,11 @@ func AddAnimation(spriteSheet:Texture, spriteFrames:SpriteFrames, animationData:
 		if "WALK" in animationData.Name && col == 0:
 			continue
 		var atlasTexture = AtlasTexture.new()
-		atlasTexture.atlas = spriteSheet
-		atlasTexture.region = Rect2(64*col,64*animationData.Row,64,64)
+		atlasTexture.atlas = spriteSheet.SpriteSheet
+		var spriteSize:int = 64
+		if spriteSheet is OversizedLPCSpriteSheet:
+			spriteSize = 192
```

Comment:
NP:

why  not avoid overwriting the value? Also you can statically type the variable without writing the name with **:=**, it grabs the type from the assigned value implicitly.
```
if spriteSheet is LPCSpriteSheet:
    var spriteSize := 64
else:
   var spriteSize := 192
```

Reply:
if I do this, the variable will be available only on the scope of the if/else. The original idea was that the class shold hold the size, but the GDScript didn't recognize that, and the class now is almost empty

It will be:
```gdscript

extends LPCSpriteSheet
class_name OversizedLPCSpriteSheet

func Size() -> int:
    return 192
```

Reply:
Understood! Out of curiosity, what kind of error / challenge are you facing placing the value directly in the class definition? Is it a nuance of plugin development?

Reply:
This error:
`res://addons/LPCAnimatedSprite/LPCAnimatedSprite2D.gd:169 - Invalid call. Nonexistent function 'Size' in base 'Resource (LPCSpriteSheet)'.`

Reply:
Just discovered. To work I needed a @tool on the script for it to run correctly on the editor